src/components/PersonLink.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Person } from '../types/Person';
+import classNames from 'classnames';
+
+type Props = {
+  person: Pick<Person, 'name' | 'sex' | 'slug'>;
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const FEMALE = person.sex === 'f';
```

Comment:
Use a `camelCase` for these variables. They are created when your component renders, so they are not constants.

Don't use magic values like `'f'` or `'m'`.